src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
+  const splitedSource = sourceString.split(';');
+
+  splitedSource
+    .map((item) => {
+      const formatedPropAndValueObject = item
+        .split(':')
+        .map((elem) => elem.trim());
+
+      return formatedPropAndValueObject;
+    })
+    .forEach(([property, value]) => {
+      Object.assign(formatedCSSData, { [property]: value });
+    });
```

Comment:
The chained methods should have the same indentation to improve readability. Align the .map and .forEach methods at the same indentation level.